tests/test_quaternion_inverse.py

```diff
@@ -0,0 +1,42 @@
+from quaternion import Quaternion
+
+def test_quaternion_inverse() -> None:
+    q = Quaternion(1,2,3,4).normalized()
+    q_inv = q.inverse()
+    identity_approx = q * q_inv # Approximately unit quaternion
+    assert (
+        abs(identity_approx.a - 1) < 1e-6 and
+        abs(identity_approx.b) < 1e-6 and
+        abs(identity_approx.c) < 1e-6 and
+        abs(identity_approx.d) < 1e-6
+    )
+
+def test_quaternion_inverse_zero() -> None:
```

Comment:
> Even if the appropriate exception is caught, it is bad style, because the test result is less informative than it would be with pytest.raises(e) (it just says pass or fail.)

В этом случае вариант с `assert False` и `try ... catch ...` работает, но для тестирования ошибок есть специальная конструкция 
```python
with pytest.raises(Exception):
    ...
```

Вариант с `try ... catch ...` менее очевидный и хуже читаемый, а ещё в нём проще допустить ошибку и он не универсальный. А что если бы требовалось проверить, что фунция вызывает `AssertionError`? Такая конструкция бы сломалась. Это я даю некоторое пояснение существованию `pytest.raises`.

Reply:
Исправил.